packages/dd-trace/src/config.js

```diff
@@ -1478,6 +1473,17 @@ class Config {
       'calculated',
```

Comment:
Bit unrelated and I think it could not happen in this case but `calculated` is not a valid origin for [telemetry](https://github.com/DataDog/instrumentation-telemetry-api-docs/blob/main/GeneratedDocumentation/ApiDocs/v2/SchemaDocumentation/Schemas/conf_key_value.md)

Reply:
It can't happen, but anyway, I'll protect it to `unknown`